src/query/functions/src/scalars/geo.rs

```diff
@@ -84,19 +83,21 @@ pub fn register(registry: &mut FunctionRegistry) {
     // init globals.
     geo_dist_init();
 
-    registry.register_passthrough_nullable_3_arg::<NumberType<F64>, NumberType<F64>, NumberType<i64>, NumberType<u64>,_, _>(
+    registry.register_passthrough_nullable_3_arg::<NumberType<F64>, NumberType<F64>, NumberType<u8>, NumberType<u64>,_, _>(
```

Comment:
Number type changed, is this intentional?

Reply:
Yup, H3 resolution should be u8, the previous type was wrong.